cdci_data_analysis/configurer.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, cfg_dict, origin=None):
                                      disp_dict['email_options']['email_sending_timeout'],
                                      disp_dict['email_options']['email_sending_timeout_default_threshold'],
                                      disp_dict['email_options']['email_sending_job_submitted'],
-                                     disp_dict['email_options']['email_sending_job_submitted_default_interval']
+                                     disp_dict['email_options'].get('email_sending_job_submitted_default_interval', 0)
```

Comment:
This interval has a default value which should be set in the configuration, and used in case it is not provided with the token. Is this for security reasons?

Reply:
It might not be set, it was not set in example I guess.  It's not wrong to have sensible defaults in some cases.
But ok I revert for consistency.